trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name > 10 ? @nameable.correct_name[0...9].to_s : @nameable.correct_name
+  end
```

Comment:
- [ ] Currently, you are comparing a string to `10`. Kindly compare the length of the string to 10. 👍